packages/x-date-pickers/src/locales/daDK.ts

```diff
@@ -0,0 +1,77 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+const daDKPickers: Partial<PickersLocaleText<any>> = {
```

Comment:
Should I leave this as Partial even though all keys are translated?

Reply:
Based on @alexfauquette opinion—we should leave the `Partial`.
The reason behind it—fewer changes are needed when we are introducing a new label. 😃